### ISSUE TITLE
'HCS_E_HYPERV_NOT_INSTALLED': undeclared identifier

### ISSUE THREAD

**Issue by CarlosNihelton** [microsoft/WSL-DistroLauncher#103]
Hello, everyone! Happy new year!

Current state of the `master` branch fails to compile with msbuild command line.
PR #101 introduced the HRESULT `HCS_E_HYPERV_NOT_INSTALLED` which seems to be defined in Windows 10 SDK, version 1809 (10.0.17763.0), a few releases behind the latest, but the launcher project file targets version 10.0.16299, thus resulting in the undeclared identifier compilation error. 

The fix seems to be simple: upgrade the target SDK to a proper version. To which version of the SDK should we upgrade to?

**Comment by Biswa96** [comment]
Obviously, the latest one. Link https://developer.microsoft.com/en-US/windows/downloads/windows-sdk/.

**Comment by CarlosNihelton** [comment]
> Obviously, the latest one. Link https://developer.microsoft.com/en-US/windows/downloads/windows-sdk/.

Thank you.

**Comment by Biswa96** [comment]
Alternatively, it will be easier just to defined the constant in source code like this:

```c
#define HCS_E_HYPERV_NOT_INSTALLED _HRESULT_TYPEDEF_(0x80370102L)
```
